default.nix

```diff
@@ -256,28 +257,14 @@ in
   deckdeckgo-starter =
     let
       sources = import ./nix/sources.nix;
-      starterKitDrv =
-        buildPackage
-          sources.deckdeckgo-starter
-          { npmCommands = [ "npm install" "npm run build" ]; };
-
-      starterKit = starterKitDrv.overrideAttrs (
-        oldAttrs: {
-          outputs = [ "out" "dist" ];
-          postInstall = "ln -s $napalm_INSTALL_DIR/dist $dist";
-        }
-      );
     in
-      pkgs.runCommand "deckdeckgo-starter" {}
-        ''
-          if [ ! -f ${starterKit.dist}/index.html ]
-          then
-            echo "Dist wasn't generated"
-            exit 1
-          else
-            touch $out
-          fi
+      buildPackage sources.deckdeckgo-starter {
+        name = "deckdeckgo-starter";
+        npmCommands = [ "npm install" "npm run build" ];
+        installPhase = ''
+          mv dist $out
```

Comment:
let's keep the check for `index.html`!

Reply:
fixed in 23035b243e972893aec33d1ab814b8ed2367d0f8